diagnostic.md

```diff
@@ -51,13 +72,24 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request DELETE http://www.example.com/donuts/2
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonuts = () {
+  return $.ajax({
+    url: http://www.example.com + '/donut/2',
```

Comment:
This will cause an error.  `url` needs to be a string and `donut` should be `donuts`.